math-scala/src/main/scala/org/apache/mahout/math/cf/SimilarityAnalysis.scala

```diff
@@ -211,9 +314,17 @@ object SimilarityAnalysis extends Serializable {
 
   }
 
-  def computeSimilarities(drm: DrmLike[Int], numUsers: Int, maxInterestingItemsPerThing: Int,
-                        bcastNumInteractionsB: BCast[Vector], bcastNumInteractionsA: BCast[Vector],
-                        crossCooccurrence: Boolean = true) = {
+  def computeSimilarities(
+    drm: DrmLike[Int],
+    numUsers: Int,
+    maxInterestingItemsPerThing: Int,
+    bcastNumInteractionsB: BCast[Vector],
+    bcastNumInteractionsA: BCast[Vector],
+    crossCooccurrence: Boolean = true,
+    minLLROpt: Option[Double] = None) = {
+
+    val minLLR = minLLROpt.getOrElse(0.0d) // accept all values if not specified
```

Comment:
i think style convention was to use 0.0 (minority split in favor 0d) but never 0.0d